master/buildbot/db/users.py

```diff
@@ -28,21 +28,23 @@ class UsDict(dict):
 class UsersConnectorComponent(base.DBConnectorComponent):
     # Documentation is in developer/db.rst
 
+    def __init__(self, *args, **kwargs):
+        super(UsersConnectorComponent, self).__init__(*args, **kwargs)
+        self._tbl = self.db.model.users
```

Comment:
Hey all. My end-of-term exams are going on right now. I'll look into this PR as soon at I'm free (3rd May).